src/cascadia/TerminalSettingsEditor/Actions.xaml

```diff
@@ -181,44 +119,267 @@
+                   TargetType="Button">
+                <Setter Property="Padding" Value="0" />
+                <Setter Property="VerticalAlignment" Value="Center" />
+                <Setter Property="Height" Value="{StaticResource EditButtonSize}" />
+                <Setter Property="Width" Value="{StaticResource EditButtonSize}" />
+            </Style>
+            <Style x:Key="AccentEditButtonStyle"
+                   BasedOn="{StaticResource AccentButtonStyle}"
+                   TargetType="Button">
+                <Setter Property="Padding" Value="3" />
+                <Setter Property="VerticalAlignment" Value="Center" />
+                <Setter Property="Height" Value="{StaticResource EditButtonSize}" />
+                <Setter Property="Width" Value="{StaticResource EditButtonSize}" />
+            </Style>
 
-        </ResourceDictionary>
-    </Page.Resources>
+            <!--  Converters  -->
+            <model:IconPathConverter x:Key="IconSourceConverter" />
+            <local:InvertedBooleanToVisibilityConverter x:Key="InvertedBooleanToVisibilityConverter" />
 
-    <ScrollViewer>
-        <StackPanel Style="{StaticResource SettingsStackStyle}">
-            <TextBlock x:Uid="Globals_KeybindingsDisclaimer"
-                       Style="{StaticResource DisclaimerStyle}" />
+            <!--  Templates  -->
+            <DataTemplate x:Key="KeyBindingTemplate"
+                          x:DataType="local:KeyBindingViewModel">
+                <ListViewItem AutomationProperties.AcceleratorKey="{x:Bind KeyChordText, Mode=OneWay}"
+                              AutomationProperties.Name="{x:Bind Name, Mode=OneWay}"
+                              Background="{x:Bind ContainerBackground, Mode=OneWay}"
+                              GotFocus="{x:Bind FocusContainer}"
+                              LostFocus="{x:Bind UnfocusContainer}"
+                              PointerEntered="{x:Bind EnterHoverMode}"
+                              PointerExited="{x:Bind ExitHoverMode}"
+                              Style="{StaticResource KeyBindingContainerStyle}">
+                    <Grid ColumnSpacing="8">
+                        <Grid.ColumnDefinitions>
+                            <!--  command name  -->
+                            <ColumnDefinition Width="*" />
+                            <!--  key chord  -->
+                            <ColumnDefinition Width="150" />
+                            <!--  edit buttons  -->
+                            <!--
+                                This needs to be 112 because that is the width of the row of buttons in edit mode + padding.
+                                3 buttons: 32+32+32
+                                Padding:       8+ 8
+                                This allows the "edit" button to align with the "cancel" button seamlessly
+                            -->
+                            <ColumnDefinition Width="112" />

```

Comment:
That's unfortunate, does it look really wonky if we left it to god and put width as auto/star sizing (maybe even scalar star sizing?)

Reply:
We're already using a `*` for the first column, so we would have to make this one proportional to that, which doesn't guarantee that the buttons will be aligned.

I tried doing 3 columns of size `auto`, but because we don't have a button for the other 2 buttons in non-edit mode, it ends up ruining the alignment. :(

Only thing I can think of is replacing the stack panel with a `Grid` with 3 `*` columns, but that seems a lot more complicated than what we have here.